.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Proposal
+about: Create a proposal for depstat
+title: ''
+labels: kind/proposal
```

Comment:
`kind/feature` and `kind/proposal` are the same thing, right?

None of the k8s repos use `kind/proposal` so I'd suggest removing this.